File: backend/app/translate/prompts/agent_txt.py
"""Phase 4 Agent TXT Prompt 入口"""

from __future__ import annotations

from .loader import load_prompt_md


def build_agent_txt_system_prompt() -> str:
    """构建 Phase 4 Agent TXT 的 System Prompt"""
    return load_prompt_md("case-4-agents-skill.md")


def build_agent_txt_user_prompt(steps_plain_text: str) -> str:
    """构建 Phase 4 Agent TXT 的 User Prompt"""
    return f"""请根据以下按时间顺序排列的底层步骤记录（纯文本），进行业务逻辑聚合并输出 <agent_chunk> XML：

{steps_plain_text}"""

File: feature_recorder/src/case_translate/prompts/agent-txt.js
/**
 * agent-txt.js - Phase 4 Agent TXT Prompt 入口
 *
 * Skill：prompts/md/case-4-agents-skill.md
 * User：本文件内嵌（动态数据拼接）
 */

import { loadPromptMd } from './loader.js';

/**
 * 构建 Phase 4 Agent TXT 的 System Prompt
 *
 * @returns {string}
 */
export function buildAgentTxtSystemPrompt() {
  return loadPromptMd('case-4-agents-skill.md');
}

/**
 * 构建 Phase 4 Agent TXT 的 User Prompt（纯文本步骤流）
 *
 * @param {string} stepsPlainText - 本窗步骤纯文本
 * @returns {string}
 */
export function buildAgentTxtUserPrompt(stepsPlainText) {
  return `请根据以下按时间顺序排列的底层步骤记录（纯文本），进行业务逻辑聚合并输出 <agent_chunk> XML：

${stepsPlainText}`;
}
